backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/ScheduledPublicationService.java

```diff
@@ -0,0 +1,75 @@
+package de.bund.digitalservice.ris.caselaw.domain;
+
+import java.time.LocalDateTime;
+import lombok.extern.slf4j.Slf4j;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class ScheduledPublicationService {
+
+  private final DocumentationUnitRepository repository;
+
+  private final HandoverService handoverService;
+
+  /**
+   * "Terminierte Abgabe": Cron job that finds doc units that have a scheduled publishing date-time
+   * now, or in the past and published them. Even if publication fails, we will unset the scheduling
+   * and save the timestamp of the last attempt.
+   */
+  public ScheduledPublicationService(
+      DocumentationUnitRepository repository, HandoverService handoverService) {
+
+    this.repository = repository;
+    this.handoverService = handoverService;
+  }
+
+  @Scheduled(fixedRateString = "PT1M") // Runs every minute
+  @SchedulerLock(name = "scheduled-publication-job", lockAtMostFor = "PT3M")
```

Comment:
Is there a reason why we use kebab-case here? The name is then written as "name" into the lock db table?

Reply:
![image](https://github.com/user-attachments/assets/2236552c-c667-4c34-8343-e8db19d1d4d6)
Yes, exactly. Would you prefer scheduledPublicationJob here? (I don't have a preference)